pkg/asset/internal/templates.go

```diff
@@ -439,7 +440,7 @@ spec:
         - --allocate-node-cidrs=true
         - --cloud-provider={{ .CloudProvider }}
         - --cluster-cidr={{ .PodCIDR }}
-        - --configure-cloud-routes=false
+        - --configure-cloud-routes=true
```

Comment:
We assume the use of flannel (or another CNI plugin) - so we do not want the cloud-provider to also be trying to configure routes.